client/index.tsx

```diff
@@ -8,6 +8,10 @@ import {AvatarContentText} from 'superdesk-ui-framework';
 import belgaImage from './belga/image';
 import belga360Archive from './belga/360archive';
 import belgaPress from './belga/belgapress';
+import moment from 'moment';
+import {IEventItem, IPlanningItem} from 'superdesk-planning/client/interfaces';
```

Comment:
For now this should be fine, but ideally we should have a public interface for the Planning module that would provide these utils and configuration abilities.

Reply:
yep didn't want to complicate it now but some public interface would be the way to go